cluster-scope/bundles/opf-grafana-public/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: opf-grafana-public
+resources:
+  - ../../base/core/namespaces/opf-grafana-public
+  - ../../base/operators.coreos.com/operatorgroups/opf-grafana-public
+  - ../../base/operators.coreos.com/subscriptions/grafana-operator
```

Comment:
I think grafana resources are getting a bit disorganized, can we clean them up as part of this pr? 

Right now the subscription is the same as [this](https://github.com/operate-first/apps/blob/master/cluster-scope/base/operators.coreos.com/subscriptions/opf-grafana/subscription.yaml) one. 

And we also have a bundle for grafana [here](https://github.com/operate-first/apps/blob/master/cluster-scope/bundles/opf-monitoring/kustomization.yaml)

Can we have these 2 bundles in their stead: 

opf-grafana
  - ../../base/core/namespaces/opf-monitoring
  - ../../base/rbac.authorization.k8s.io/clusterrolebindings/opf-monitoring-grafana-cluster-view
  - grafana-serviceaccount.yaml
  - ../../base/operators.coreos.com/subscriptions/grafana-operator

opf-grafana-public 
  - ../../base/core/namespaces/opf-grafana-public
  - ../../base/operators.coreos.com/operatorgroups/opf-grafana-public
  - ../../base/operators.coreos.com/subscriptions/grafana-operator

what do you think? would that be better?

Reply:
hrmm I'm realizing now that would not work, we don't use `opf-monitoring` bundling to only install the operator

Reply:
okay for now just use the same `subscription` [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/operators.coreos.com/subscriptions/opf-grafana/subscription.yaml) in both places, since the bundle kustomization will update the namespace. That should at least reduce duplication.

Reply:
```
../../base/core/namespaces/opf-monitoring
../../base/rbac.authorization.k8s.io/clusterrolebindings/opf-monitoring-grafana-cluster-view
```
These are independent of the grafana operator so I have made it as a separate bundle. Plus it doesn't make sense to restructure unrelated stuff in this PR, we can make a separate PR for that.

Reply:
> okay for now just use the same subscription here in both places, since the bundle kustomization will update the namespace. That should at least reduce duplication.

yep that is how it is rn.